src/arch/riscv/RiscvMMU.py

```diff
@@ -49,7 +49,7 @@ class RiscvMMU(BaseMMU):
 
     itb = RiscvTLB(entry_type="instruction")
     dtb = RiscvTLB(entry_type="data")
-    pma_checker = Param.PMAChecker(PMAChecker(), "PMA Checker")
+    pma_checker = Param.BasePMAChecker(PMAChecker(), "PMA Checker")
     pmp = Param.PMP(PMP(), "Physical Memory Protection Unit")
```

Comment:
General note, not a specific comment on the PR. There's no need to take any action on this.

I think we need to move away from the design where we instantiate SimObjects in these SimObject descriptions. It's super error prone and causes significant confusion when trying to find the "default" values for SimObject parameters.

Reply:
I think I will keep the default here to make the config script backward compatable

Reply:
Yeah, this make sense. At some point in the future, I would like to start fixing these things, though :)